challenges-14-plus.yml

```
@@ -0,0 +1,472 @@
+asset-tracking:
+  question: |
+    I just came from our company meeting and I have big news! We have just released a
+    new version of our main CSS file!
+    
+    But I'm getting complaints from users who are unable to see those changes! Oh no!
+    
+    How can I make Turbo Drive show the new version of my site to all my users without
+    waiting for them to do a full refresh (i.e. by closing their browsers and come back
+    the next day)?
+  explanation: |
+    One of the few instances in which we can say that a full reload is *needed* is when
+    our front end code changes (JavaScript or CSS)!
```

```suggestion
    our frontend code changes (JavaScript or CSS)!
```

This can be spelled so many ways, like `front-end`, but this is the one I like :)